src/main/resources/grel_java_mapping.ttl

```diff
@@ -155,6 +155,20 @@ grelm:array_uniquesMapping
                            fnom:method-name "uniques" ] ;
 .
 
+#SUM
```

Comment:
Can you move it above 'Uniques', so that it's synced with https://github.com/FnOio/grel-functions-fno/blob/master/grel.ttl ?